mitodl/micromasters
# Updated UX for education signup screen

#### What are the relevant tickets?

Closes #929 

#### What's this PR do?

Applies various changes to the education page of the signup flow (see issue for details)

#### Where should the reviewer start?

Probably static/js/components/EducationForm.js

#### How should this be manually tested?

Set filled_out=false for one of your users and go to the second page of signup. Observe the behavior of the Yes/No radio buttons and entry deletion.

#### Screenshots (if appropriate)

![ss 2016-09-07 at 18 24 31](https://cloud.githubusercontent.com/assets/14932219/18330746/5bf6a3a2-7528-11e6-9f16-28faf966c830.png)
